### PR TITLE
Perf tests for all supported codecs against Float64 and UInt64.

### PR DESCRIPTION
**Pull request by Enmk**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):
Performance tests for all codecs against Float64 and UInt64 values (where applicable).
...

Detailed description (optional):

...

### CODE REVIEW

File: dbms/tests/performance/codecs_float.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <value>rnd</value>
+            </values>
+        </substitution>
+    </substitutions>
+
+    <create_query>CREATE TABLE IF NOT EXISTS codec_{seq_type}_{type}_{codec} (n {type} CODEC({codec})) ENGINE = MergeTree PARTITION BY tuple() ORDER BY tuple();</create_query>
+
+    <!-- Using limit to make query finite, allowing it to be run multiple times in a loop, reducing mean error -->
+    <query>INSERT INTO codec_seq_Float64_{codec} (n) SELECT number/pi() FROM system.numbers LIMIT 100000 SETTINGS max_threads=1</query>
+    <query>INSERT INTO codec_mon_Float64_{codec} (n) SELECT number+sin(number) FROM system.numbers LIMIT 100000 SETTINGS max_threads=1</query>
```
Comment (alexey-milovidov) [inline dbms/tests/performance/codecs_float.xml]: What is the meaning of "mon"?

File: dbms/tests/performance/codecs_float.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <query>INSERT INTO codec_rnd_Float64_{codec} (n) SELECT (rand() - 4294967295)/pi() FROM system.numbers LIMIT 100000 SETTINGS max_threads=1</query>
```
Comment (alexey-milovidov) [inline dbms/tests/performance/codecs_float.xml]: It's strange that we have all non-positive values but still Ok.

File: dbms/tests/performance/codecs_float.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    <!-- INSERTs above will be run unspecified amount of times, hence size of table is unknown.
+    To make test more reliable, we SELECT fixed number of rows. -->
+    <query>SELECT count(n) FROM codec_{seq_type}_{type}_{codec} WHERE ignore(n) LIMIT 100000 SETTINGS max_threads=1</query>
```
Comment (alexey-milovidov) [inline dbms/tests/performance/codecs_float.xml]: It will depend on whether merge is processing right now or not.

Comment (Enmk) [reply dbms/tests/performance/codecs_float.xml]: Fixed, split into SELECT and INSERT tests

File: dbms/tests/performance/codecs_float.xml (hunk 2 above)
Comment (Enmk) [reply dbms/tests/performance/codecs_float.xml]: Any kind of sequences can be added later, this is just a starting point.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**